data-serving/data-service/schemas/cases.schema.json

```diff
@@ -213,110 +213,125 @@
         }
       },
       "travelHistory": {
-        "bsonType": "array",
-        "uniqueItems": true,
-        "items": {
-          "bsonType": "object",
-          "additionalProperties": false,
-          "properties": {
-            "_id": {
-              "bsonType": "objectId"
-            },
-            "location": {
+        "bsonType": "object",
+        "additionalProperties": false,
```

Comment:
Is this true anywhere? If not, can we inherit from the top-level schema?

Reply:
unfortunately, it doesn't seem to inherit :\ I tested removing it from a sub-schema and adding a random prop to that schema and it validates. I can't find any documentation on it though :(